jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -261,6 +261,8 @@ object ConverterUtils extends Logging {
           typedFuncName.concat("date")
         case StringType =>
           typedFuncName.concat("str")
+        case DecimalType() =>
+          typedFuncName.concat("decimal")
```

Comment:
Maybe change to "dec" for decimal to be consistent with Substrait upstream.
Reference: https://substrait.io/extensions/#function-signature-compound-names